hack/scripts/generate_crds.sh

```diff
@@ -0,0 +1,68 @@
+#!/bin/bash
+set -eo pipefail
+
+# This bash is the replacement for: ./boilerplate/_lib/container-make op-generate
+#                               or:  make -n generate
```

Comment:
You're only overriding `op-generate` in your Makefile, but you're providing the guts of `go-generate` and `openapi-generate` in this script. Seems those could have been omitted, yes?